README.md

```diff
@@ -1,2 +1,63 @@
-# worldMap-countryStats
-A web application connected to the API shows world map and country stats 
+# Word Map and Country Stats
+
+### 📜
+- This web application the world map with contries Stats I used an API to get the countries stats flags and and maps.
+
+ a web site that has all information about space staion Rokets Missions and Dragons. 
+
+### 💻 Live DEMO
+
+   [Online view](https://thunderous-klepon-eb393d.netlify.app/)
+   
+   [Video](https://www.loom.com/share/70e5318c28b6424d8ddffccdb548eec3)
+
+## 🏛 Built With
+- css
+- JavaScript
+- React
+- Redux
+- ES6 Modules
+- following linters standards.
+- following github flows standards.
+- following git flows standards.
+
+## 📑 Authors
+👤 **Ahmed Zaki Yousufi**
+- GitHub: [@Ahmed Zaki Yousufi](https://github.com/zakiyousufi)
+- LinkedIn: [Ahmed Zaki Yousufi](https://www.linkedin.com/in/ahmadzaki-yousufi-055214217/)
+
+## ⚙ Getting Started
+To get a local copy up and running... 
+Follow these 
+
+  ## 📋Setup
+- Run git clone on this project at the desired directory:
+
+  git clone https: CLONE URL (You can get the URL from: [(https://github.com/zakiyousufi/worldMap-countryStats)](github.com/zakiyousufi/worldMap-countryStats) Click on the code button at the top of the repo (it has green color) then select the HTTPS or ssh key and copy the URL)
+  ```
+  git clone (Past the URL that you copied)
+  ```
+  ```
+  cd worldMap-countryStats
+  ```
+✅Install dependencies
+
+  ```
+  npm install
+  ```
+
+✅Start React Dev Server
+  ```
+  npm start
+  ```
+- On your favourite browser go to http://localhost:3000/
+
+## 🤝 Contributing
+Contributions, issues, and feature requests are welcome!
+thanks:
+
+## 🙏🏻 Show your support
+Give a ⭐️ if you like this project!
+
+## 📝 License
+This project is [MIT](./MIT.md) licensed.
```

Comment:
The link to your license is broken. Kindly use the right link to your license since you have it in your root directory or otherwise remove this section.

- It is important that all links in your README work so it doesn't come across as sloppy and unfinished.

Reply:
@KDlamini Thanks